Build Task: Error when running `react:build` gulp task

<!-- Thanks for contributing to Jetpack!
Pick a clear title ("Sharing: add new Facebook button") and proceed. -->

#### Steps to reproduce the issue

1. Clone repo
1. Run `yarn build` from root of repo on `master` branch

#### What I expected
<!-- A clear and concise description of what you expected to happen. -->

Build development assets

#### What happened instead

An error:

```
[13:13:37] 'react:master' errored after 12 s
[13:13:37] ModuleNotFoundError in plugin "webpack"
Message:
    Module not found: Error: Can't resolve 'history/lib/Actions' in '/Users/Grant/Automattic/Sites/jetpack/node_modules/react-router/es'
Details:
    details: resolve 'history/lib/Actions' in '/Users/Grant/Automattic/Sites/jetpack/node_modules/react-router/es'
```

This appears to be caused by updating `history` package to v4 (https://github.com/Automattic/jetpack/commit/8fe1ddd89c62cc37dd592a3819c61172b07ac885)

Reverting that commit allowed me to complete the build task successfully.

<!--
PLEASE NOTE
- Everything is optional, but try to add as many details as possible.
- If requesting a new feature, explain why you'd like to see it added.
- This issue tracker is not for support. If you have questions about Jetpack, you can [start a new thread in the Jetpack support forums](https://wordpress.org/support/plugin/jetpack#new-post), or [send us an email](http://jetpack.com/contact-support/).
- Do not report potential security vulnerabilities here. For responsible disclosure of security issues and to be eligible for our bug bounty program, please submit your report via [the HackerOne portal](https://hackerone.com/automattic).
-->

--- a/_inc/client/state/jumpstart/actions.js
+++ b/_inc/client/state/jumpstart/actions.js
@@ -3,7 +3,7 @@
  */
 import { createNotice, removeNotice } from 'components/global-notices/state/notices/actions';
 import { translate as __ } from 'i18n-calypso';
-import { createBrowserHistory } from 'history';
+import { createHistory } from 'history';
 import analytics from 'lib/analytics';
 
 /**
@@ -57,7 +57,7 @@ export const jumpStartActivate = () => {
 	};
 };
 
-const history = createBrowserHistory();
+const history = createHistory();
 
 export const jumpStartSkip = () => {
 	return ( dispatch ) => {

--- a/package.json
+++ b/package.json
@@ -107,7 +107,7 @@
     "gulp-sourcemaps": "2.6.4",
     "gulp-tap": "0.1.3",
     "gulp-uglify": "3.0.1",
-    "history": "4.7.2",
+    "history": "3.3.0",
     "i18n-calypso": "3.0.0",
     "jsdom": "9.12.0",
     "jsdom-global": "2.1.1",

